src/cloud-providers/gcp/provider.go

```diff
@@ -0,0 +1,191 @@
+// (C) Copyright Confidential Containers Contributors
```

Comment:
Any plans to add unit tests and e2e tests as follow-on PRs ?

Reply:
Tests were part of the provisioning code, and we removed from this PR to wait for the GKE fix. But is currently implemented here:

https://github.com/confidential-containers/cloud-api-adaptor/compare/main...beraldoleal:cloud-api-adaptor:gcp-plus-ueda

I just need to check again and enable more test.

So, my plan was to add a follow up PR adding the provisioning + tests after this PR gets merged. Let me know if you agree with that.

Reply:
Makes sense to me. The tests can be added as a follow-on PR.